ui-backoffice/src/content/pages/Management/Partners/Members/BulkActions.tsx

```diff
@@ -0,0 +1,94 @@
+import { useState, useRef } from 'react';
+
+import {
+  Box,
+  Menu,
+  IconButton,
+  Button,
+  ListItemText,
+  ListItem,
+  List,
+  Typography
+} from '@mui/material';
+import { styled } from '@mui/material/styles';
+
+import DeleteTwoToneIcon from '@mui/icons-material/DeleteTwoTone';
+import MoreVertTwoToneIcon from '@mui/icons-material/MoreVertTwoTone';
+
+const ButtonError = styled(Button)(
+  ({ theme }) => `
+     background: ${theme.colors.error.main};
+     color: ${theme.palette.error.contrastText};
+
+     &:hover {
+        background: ${theme.colors.error.dark};
+     }
+    `
+);
+
+function BulkActions() {
+  const [onMenuOpen, menuOpen] = useState<boolean>(false);
```

Comment:
Il vaut mieux prefixer par `set` la function setter des useState

```suggestion
const [isMenuOpen, setIsMenuOpen] = useState<boolean>(false);
```

Reply:
Oui, c'est pas mon code, j'adapte seulement le template à nos besoins pour le moment j'ai pas eu le temps de tout passer au peigne fin